### PR TITLE
Image loading from url

### PR DESCRIPTION
**Pull request by cindyh25**

### CODE REVIEW

File: app/src/main/java/com/example/flick/ListActivity.kt [hunk 1]
```diff
@@ -138,71 +140,3 @@ class ListActivity: AppCompatActivity() {
...
-        return data.size
-    }
-
-    override fun onBindViewHolder(holder: TagsViewHolder, position: Int) {
-        val tag: Tag = data[position]
-        holder.bind(tag)
-        holder.itemView.setOnClickListener {
-            holder.itemView.setBackgroundResource(R.drawable.list_tag_active)
-            // filter list
-            // listMedia.filter { m -> m.tags.id == tag.id }
```
Comment (alanna-zhou) [inline app/src/main/java/com/example/flick/ListActivity.kt:191]: did you mean to delete this xD

File: app/src/main/java/com/example/flick/ListTagsAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        return data.size
+    }
+
+    override fun onBindViewHolder(holder: TagsViewHolder, position: Int) {
+        val tag: Tag = data[position]
+        holder.bind(tag)
+        holder.itemView.setOnClickListener {
+            holder.itemView.setBackgroundResource(R.drawable.list_tag_active)
+            // filter list
+            // listMedia.filter { m -> m.tags.id == tag.id }
```
Comment (alanna-zhou) [inline app/src/main/java/com/example/flick/ListTagsAdapter.kt:25]: did you mean to delete this xD

File: app/src/main/java/com/example/flick/models/MediaList.kt [hunk 3]
```diff
@@ -3,5 +3,5 @@ package com.example.flick.models
 data class MediaList(val lstId: Int, val lstName: String,
                      val lstPic: String?, val isFavorite: Boolean,
```
Comment (alanna-zhou) [inline app/src/main/java/com/example/flick/models/MediaList.kt:4]: should this be changed to `is_saved` and `is_watch_later`?

File: app/src/main/java/com/example/flick/ListTagsAdapter.kt (hunk 2 above)
Comment (cindyh25) [reply app/src/main/java/com/example/flick/ListTagsAdapter.kt:25]: oopsies yeah

File: app/src/main/java/com/example/flick/models/MediaList.kt (hunk 3 above)
Comment (cindyh25) [reply app/src/main/java/com/example/flick/models/MediaList.kt:4]: yes thanks for catching!

**Pull request merged**